specification/mediaservices/resource-manager/Microsoft.Media/stable/2020-05-01/streamingservice.json

```diff
@@ -1373,52 +1539,95 @@
       "properties": {
         "description": {
           "type": "string",
-          "description": "The Live Event description."
+          "description": "A description for the live event."
         },
         "input": {
           "type": "object",
           "$ref": "#/definitions/LiveEventInput",
-          "description": "The Live Event input."
+          "description": "Live event input settings. It defines how the live event receives input from a contribution encoder."
         },
         "preview": {
           "type": "object",
           "$ref": "#/definitions/LiveEventPreview",
-          "description": "The Live Event preview."
+          "description": "Live event preview settings. Preview allows live event producers to preview the live streaming content without creating any live output."
         },
         "encoding": {
           "type": "object",
           "$ref": "#/definitions/LiveEventEncoding",
-          "description": "The Live Event encoding."
+          "description": "Encoding settings for the live event. It configures whether a live encoder is used for the live event and settings for the live encoder if it is used."
+        },
+        "transcriptions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LiveEventTranscription"
+          },
+          "description": "Live transcription settings for the live event. See https://go.microsoft.com/fwlink/?linkid=2133742 for more information about the live transcription feature."
```

Comment:
I like the use of FWLink to the article here.  We should look for other areas to do that in our descriptions where necessary for clarity.